components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -666,7 +666,7 @@ this mask will have smb calculated over the entire global land surface
 <stream_year_first_ndep use_cn=".true."   sim_year="constant" sim_year_range="2000-2100" >2000</stream_year_first_ndep>
 <stream_year_last_ndep  use_cn=".true."   sim_year="constant" sim_year_range="2000-2100" >2100</stream_year_last_ndep>
 
-<stream_fldfilename_ndep hgrid="1.9x2.5"   use_cn=".true."   >lnd/clm2/ndepdata/fndep_elm_cbgc_exp_simyr1849-2101_1.9x2.5_c190103.nc</stream_fldfilename_ndep>
+<stream_fldfilename_ndep hgrid="1.9x2.5"   use_cn=".true."   >lnd/clm2/ndepdata/fndep_elm_cbgc_exp_simyr1849-2101_1.9x2.5_ssp245_c240903.nc</stream_fldfilename_ndep>
```

Comment:
@bishtgautam , don't know why hgrid attribute is specified and if it is needed. It has been tested it serves as a general default file independent of grid resolution. Therefore, no value is specified in the use_case file for the 1850, 2010 and 20TR compsets.

Reply:
@wlin7, I agree that the `hgrid` attribute is unnecessary and should be removed for all the Ndep dataset entries.